dbaas/logical/templates/logical/database/details/credentials_tab.html

```diff
@@ -202,8 +202,13 @@ <h3 class="panel-title">SSL</h3>
     django.jQuery(function() {
       {% if database.engine.has_users %}
         {% for credential in database.credentials.all %}
+          var checked = ""
+          {% if credential.force_ssl %}
+            var checked = "checked"
```

Comment:
keywork var is not necessary here

Reply:
or 
var checked = {% if credential.force_ssl %} "checked" {% else %} "" {% endif %};